olp-cpp-sdk-core/include/olp/core/http/Network.h

```diff
@@ -90,7 +105,24 @@ class CORE_API Network {
    *
    * @param headers The default headers.
    */
-  virtual void SetDefaultHeaders(Headers) {}
+  virtual void SetDefaultHeaders(Headers headers);
+
+  /**
+   * @brief Set the current statistics bucket.
+   *
+   * @param[in] bucked_id The bucket ID.
+   */
+  virtual bool SetCurrentBucket(uint8_t bucket_id);
+
+  /**
+   * @brief Get the statistics for bucket.
+   *
+   * By default returns statistics for a default bucket.
+   *
+   * @param[in] bucked_id The bucket ID.
+   * @return Statistics which contains the statistic for a requested bucket.
```

Comment:
```suggestion

   * @return Statistics which contains the statistic for the requested bucket.
```
NOTE: Extra newline between parameters and return.